### ISSUE TITLE
ocgrpc.NewClientStatsHandler

### ISSUE THREAD

**Issue by naguibihab** [GoogleCloudPlatform/google-cloud-go#938]
## Client

PubSub

## Describe Your Environment

Local Golang

## Expected Behavior

To build

## Actual Behavior

I get this error when I run `go get`: `ocgrpc.NewClientStatsHandler`

## Issue

I am following this tutorial to publish a topic to Pub/Sub from a golang project and here's the code I have for that project at the moment:

```
package main

import "cloud.google.com/go/pubsub"
import "fmt"

func main() {
    fmt.Printf("hello, world\n")
}
```

All it does is simply imports the pubsub but when I run `go get` I get this error: `undefined: ocgrpc.NewClientStatsHandler`

```
C:\Users\iha001\Dev\golang-projects\src\github.com\naguibihab\golang-playarea\src\gcloud>go get
# cloud.google.com/go/pubsub
..\..\..\..\..\cloud.google.com\go\pubsub\go18.go:34:51: undefined: ocgrpc.NewClientStatsHandler
```

**Comment by naguibihab** [comment]
It seems that windows doesn't go well with golang: https://stackoverflow.com/a/49355423/830230

**Comment by PedroBrochado** [comment]
@naguibihab This is NOT a windows issue. This commit fixes the problem be072a5d1d73144ae0ce1071e9bd43d1ad221581.
Short explanation: breaking changes where pushed on a minor release of a google pubsub dependency: https://github.com/census-instrumentation/opencensus-go/commit/ac824559a1df1b4a64405c4321f820c58b498e91, method `NewClientStatsHandler` was deleted. (They don't claim anywhere they comply with semver).

**Comment by jadekler** [comment]
Hi @PedroBrochado - I believe we resolved this issue in https://github.com/GoogleCloudPlatform/google-cloud-go/commit/be072a5d1d73144ae0ce1071e9bd43d1ad221581. Could you try pulling from HEAD?
